scripts/scil_prepare_topup_command.py

```diff
@@ -97,40 +97,52 @@ def main():
 
     gtab = gradient_table(bvals, bvecs, b0_threshold=b0_threshold)
 
-    acqparams = create_acqparams(gtab, args.readout, args.encoding_direction)
+    acqparams = create_acqparams(gtab, args.readout, args.encoding_direction,
```

Comment:
create_acqparams should take number of b0s AP and number of PA directions as args.

You can do that later in the code. L127 ish.